Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -88,6 +96,10 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager, El
         masterTalons = new HashMap<XCANTalon, BaseDriveSubsystem.MotionRegistration>();
         masterTalons.put(leftMaster, new MotionRegistration(0, 1, -1));
         masterTalons.put(rightMaster, new MotionRegistration(0, 1, 1));
+        
+        this.leftPidManager = pf.createPIDManager(getPrefix()+"Drive velocity (local)", 0, 0, 0, 0, 1, -1);
```

Comment:
You should remove the word `Drive` here since it's already in the "/Drive/" network table.